src/rest-api/content-helper/class-content-helper-controller.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * API Content Helper Controller
+ *
+ * @package Parsely
+ * @since   3.17.0
+ */
+
+declare(strict_types=1);
+
+namespace Parsely\REST_API\Content_Helper;
+
+use Parsely\REST_API\REST_API_Controller;
```

Comment:
The code itself isn't complicated, though here we already have some deeply nested inheritance (`Base_API_Controller -> REST_API_Controller -> Content_Helper_Controller`).

Just wanted to poke your brain about why this approach was chosen (e.g. no alternatives, easier testing, etc.) and whether any alternatives were considered.

Reply:
The idea is to have each namespace (`content-helper`, `stats`, and `settings`) being its own "isolated" REST API, so they can each register their own endpoints.

The REST API controller will pretty much initialize and execute each individual controller, but also includes some common parameters that are shared by all the other controllers - such as the global namespace (`wp-parsely`) and version (`v2`).

This effectively allows the initialization of the REST API to happen in only a couple lines, and we can avoid going to multiple files to initialize a new endpoint, for example.

```php
// Initialize the REST API Controller.
$rest_api_controller = new REST_API_Controller( $GLOBALS['parsely'] );
$rest_api_controller->init();
```